File: src/main/java/com/soutvoid/tvpr/TvprApplication.java
package com.soutvoid.tvpr;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@ComponentScan("com.soutvoid")
@SpringBootApplication
public class TvprApplication {

        public static void main(String[] args) {
            SpringApplication.run(TvprApplication.class, args);
        }

}

File: src/main/kotlin/com/soutvoid/tvpr/TvprApplication.kt
package com.soutvoid.tvpr

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.ResponseBody

@SpringBootApplication
class TvprApplication


fun main(args: Array<String>) {
    SpringApplication.run(TvprApplication::class.java, *args)
}
